tooling/components/io.siddhi.distribution.editor.core/src/main/resources/web/js/export-deployment-artifacts/template-file-dialog.js

```diff
@@ -19,21 +19,22 @@
 define(['require', 'lodash', 'jquery', 'log', 'ace/ace', 'app/source-editor/editor'],
     function (require, _, $, log, ace, SiddhiEditor) {
 
+        // todo rename to template-app-dialog.js
         var TemplateFileDialog = function (options) {
             this.app = options.app;
             this.appNames = options.siddhiAppNames;
-            this.templateContainer = options.templateHeader;
-            this.appArr = this.readSiddhiApps(this.appNames);
-            this.editorObjArr = [];
+            this.templateContainer = options.templateContainer;
+            this.appObjectArrayList = this.readSiddhiApps(this.appNames);
+            this.editorObjectArrayList = [];
         };
 
         TemplateFileDialog.prototype.constructor = TemplateFileDialog;
 
         TemplateFileDialog.prototype.render = function () {
             var self = this;
             var i;
-            for (i = 0; i < self.appArr.length; i++) {
-                var entry = self.appArr[i];
+            for (i = 0; i < self.appObjectArrayList.length; i++) {
```

Comment:
better to use loadash js for loops , check in fill template file dialog

Reply:
@pcnfernando FYA

Reply:
Fixed in 5a69e9a